### ISSUE TITLE
      [Accessibility : Alternative content] Input field in Add volume page does not have alternative content 

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/kimchi#889]
Input fields in Add volume page does not have alternative content.
